web/vue.config.js

```diff
@@ -0,0 +1,59 @@
+const path = require('path')
+const appType = process.env.VUE_APP_TYPE
+
+function resolve(dir) {
+  return path.join(__dirname, dir)
+}
+
+function getProxyTargetPort() {
+  switch (appType) {
+    case 'frps':
+      return 8081
+    case 'frpc':
+      return 8082
+    default:
+      return 8080
+  }
```

Comment:
Actually, this requires frpc & frps to listen the specified port as dashboard's port, so could you make a change:
`8081 -> 7500 & 8082 -> 7400`, because some users set config file from [frpc_full.ini]( https://github.com/fatedier/frp/blob/dev/conf/frpc_full.ini#L52) or [frps_full.ini](https://github.com/fatedier/frp/blob/dev/conf/frps_full.ini#L36).